test/parallel/test-bootstrap-modules.js

```diff
@@ -73,7 +73,6 @@ const expectedModules = new Set([
   'NativeModule internal/modules/cjs/helpers',
   'NativeModule internal/modules/cjs/loader',
   'NativeModule internal/modules/esm/assert',
-  'NativeModule internal/modules/esm/create_dynamic_module',
```

Comment:
And you’ll delete this file once this PR isn’t draft? Or can we not delete it yet because it’s still used by something?

Reply:
So shouldn’t it be deleted in this PR, rather than in a follow-up PR?